e2e/tests/ui-driven/src/create-flow.spec.ts

```diff
@@ -158,15 +168,7 @@ test.describe("Flow creation, publish and preview", () => {
 
     await page.goto(`/${context.team.slug}/${serviceProps.slug}`);
 
-    // Open flow settings
-    page.locator('[aria-label="Service settings"]').click();
-
-    // Toggle flow online
-    page.getByLabel("Offline").click();
-    page.getByRole("button", { name: "Save", disabled: false }).click();
-    await expect(
-      page.getByText("Service settings updated successfully"),
-    ).toBeVisible();
+    await turnServiceOnline(page);
```

Comment:
Nice, I highly approve of extracting out these helpers! 👍

Reply:
Could the previous line `await page.goto(`/${context.team.slug}/${serviceProps.slug}`);` be included in these helpers?

Reply:
I think there are other instances in the test where `navigateToService` could be used, e.g. line 121

Reply:
Yeah good shout, I was treading quite lightly with this.